source/community/reactnative/src/components/containers/container-wrapper.js

```diff
@@ -47,15 +60,125 @@ export class ContainerWrapper extends React.PureComponent {
     getBackgroundImageContainer = () => {
         if (Utils.isString(this.payload.backgroundImage)) {
             this.payload.backgroundImage = {
-                url: this.payload.backgroundImage
-            }
+                url: this.payload.backgroundImage,
+            };
         }
+        // padding
+		const padding = this.props.configManager.hostConfig.getEffectiveSpacing(Enums.Spacing.Padding);
         return (
             <React.Fragment>
-                <BackgroundImage backgroundImage={this.payload.backgroundImage} />
-                {this.props.children}
-            </React.Fragment >
+                <BackgroundImage
+                    backgroundImage={this.payload.backgroundImage}
+                />
+                <View style={{paddingTop: padding, paddingBottom: padding}}>
```

Comment:
Again can be replaced with paddingVertical.

Reply:
Resolved